### ISSUE TITLE
decay_factor does not take any effect in DCH

### ISSUE THREAD

**Issue by liut0012** [thulab/DeepHash#11]
I found it is due to: 
In DeepHash/model/dch/dch.py, line 140
`lr = tf.train.exponential_decay(self.lr, global_step, self.decay_step, self.lr, staircase=True)`

I think the second learning rate `self.lr` should be `self.decay_factor`

Hope this helps in some way

**Comment by bl0** [comment]
Thanks! 
Can you send a PR for this issue?

**Comment by bl0** [comment]
Thanks very much.
